src/webview/components/Atoms/ToolTip/ToolTip.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useRef, useEffect } from 'react';
+import Icon from '../Icon/Icon';
+import './ToolTip.css';
+
+interface ToolTipProps {
+  text: string;
+}
+
+const ToolTip = ({ text }: ToolTipProps) => {
+  const [isHovered, setIsHovered] = useState(false);
+  const [position, setPosition] = useState<'right' | 'left'>('right');
+  const [maxWidth, setMaxWidth] = useState(200);
+  const containerRef = useRef<HTMLDivElement>(null);
+  const tooltipRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
```

Comment:
I added this long list of checks so that it doesn't go over the side of the screen. sadly out things can't 'pop out' of the sidebar so I just reverse it if necessary.

Reply:
i see. I tested it. I looks great. 

Just minor issue with the border line of the '?' abit cutting off from when the width is at 339 px until 356 px. I think you could shift it to the left instead of right until 356 px to make it not too close to the side of the screen. But that is just my preference/suggestion.

Reply:
I also noticed the tab bar line size is different compared to the second bar line below the paw.

Reply:
oh yea the paw liquid colour is reversed XXXDD